framework/test/mocha/integration/delegates/synchronous_tasks.js

```diff
@@ -45,8 +45,6 @@ describe.skip('integration test (delegates) - synchronous tasks', () => {
 			durationMs = intervalMs + 1;
 
 			before(done => {
-				library.modules.rounds.onBlockchainReady =
```

Comment:
Dpos module implements a replacement function event for `onBlockchainReady`?

Reply:
It was only used to return whether the Rounds module was loaded or not: `__private.loaded = true;`

Reply:
It was an event. Previously, (perhaps long time ago) this event was listened for in other parts of the application. For example, the application waiting for this event before accepting HTTP requests. Is that no longer the case?

Reply:
I couldn't find any use of it anymore. I reckon it's safe to remove it.

Reply:
Yes, I think it's safe to remove if I recall correctly this was a left over from tech-debt effort and was left there to be removed.